otter/test/cloud_client/test_rcv3.py

```diff
@@ -308,16 +344,63 @@ def test_retries(self):
         self.assertEqual(
             perform_sequence(seq, r.bulk_delete(pairs)), success_resp)
 
+    def test_retries_uppercase(self):
```

Comment:
If the *only* difference is uppercase/lowercase, the test case should be parameterized so that we don't have so much duplication of code.